lib/player.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Player
+  attr_accessor :name, :weapon
```

Comment:
**[REQUIRED]**
Since both `name` and `weapon` are not going to be changed after been set, we can change attr_accessor to attr_reader. Encapsulation is important because it is a protective shield that prevents data from being accessed and / or altered by code outside that shield.